apps/web/src/app/(admin)/backoffice/page.tsx

```diff
@@ -1,196 +1,6 @@
-
-  const handleSubmit = (ev: FormEvent<HTMLFormElement>) => {
-    ev.preventDefault() // This line prevents the default form submission
-
-    const formData = new FormData(ev.currentTarget)
-    const name = formData.get('name') as string
-    const description = formData.get('description') as string
-    const prompt = formData.get('prompt') as string
-
-    create({
-      name,
-      description,
-      prompt,
-      configuration,
-    })
-  }
-
-  const {
-    configuration,
-    handleTypeChange,
-    handleRangeFromChange,
-    handleRangeToChange,
-  } = useEvaluationConfiguration()
-
-  return (
-    <div className='w-full flex flex-col gap-4'>
-      <form onSubmit={handleSubmit}>
-        <FormWrapper>
-          <FormField label='Name'>
-            <Input
-              required
-              name='name'
-              placeholder='Enter title'
-              className='w-full'
-            />
-          </FormField>
-          <FormField label='Description'>
-            <TextArea
-              required
-              name='description'
-              minRows={4}
-              maxRows={6}
-              placeholder='Describe what is the purpose of this template'
-              className='w-full'
-            />
-          </FormField>
-          <FormField label='Prompt'>
-            <TextArea
-              required
-              name='prompt'
-              minRows={4}
-              maxRows={6}
-              placeholder='Write your template prompt'
-              className='w-full'
-            />
-          </FormField>
-          <TabSelector
-            options={[
-              { label: 'Text', value: EvaluationResultableType.Text },
-              { label: 'Number', value: EvaluationResultableType.Number },
-              { label: 'Boolean', value: EvaluationResultableType.Boolean },
-            ]}
-            onSelect={handleTypeChange}
-            selected={configuration.type}
-          />
-          {configuration.type === EvaluationResultableType.Number && (
-            <FormField label='Range'>
-              <div className='flex flex-row items-center flex-1 gap-4'>
-                <Input
-                  type='number'
-                  min={0}
-                  value={configuration.detail?.range.from.toString() || ''}
-                  placeholder='From'
-                  onChange={handleRangeFromChange}
-                />
-                <Input
-                  type='number'
-                  min={0}
-                  value={configuration.detail?.range.to.toString() || ''}
-                  placeholder='To'
-                  onChange={handleRangeToChange}
-                />
-              </div>
-            </FormField>
-          )}
-          <Button type='submit'>Create Template</Button>
-        </FormWrapper>
-      </form>
-    </div>
-  )
+export default async function AdminPage() {
+  redirect(ROUTES.backoffice[BackofficeRoutes.templates].root)
```

Comment:
Not to change now. But redirects cause errors in NextJS https://github.com/latitude-dev/latitude-llm/issues/285

Is by design but I think we need to do different